pkg/offline/offline_test.go

```diff
@@ -950,6 +944,25 @@ func TestQueue_PushMany(t *testing.T) {
 	assert.JSONEq(t, string(dataJs), stored[2].Heartbeat)
 }
 
+func TestQueue_Count(t *testing.T) {
+	// setup
+	db, cleanup := initDB(t)
+	defer cleanup()
+
+	tx, err := db.Begin(true)
+	require.NoError(t, err)
+
+	q := offline.NewQueue(tx)
+	q.Bucket = "test_bucket"
+
+	count, err := q.Count()
+	require.NoError(t, err)
+	assert.Equal(t, count, 0)
```

Comment:
Here it would be interesting to test it against an offline queue with heartbeats inside.